ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/FailureMessage.java

```diff
@@ -35,6 +36,15 @@ public static PubsubMessage of(Object caller, String payload, Throwable e) {
     return FailureMessage.of(caller, payload.getBytes(StandardCharsets.UTF_8), e);
   }
 
+  /**
+   * Return a PubsubMessage wrapping a SponsoredInteraction with attributes describing the error.
+   */
+  public static PubsubMessage of(Object caller, SponsoredInteraction interaction, Throwable e) {
```

Comment:
In terms of modules, it's not great that we're importing from `contextualservices` in this higher-level, but don't have a better suggestion for this.

Reply:
No action needed on this. Might be worth calling as a todo in the comments.